packages/flutter_migrate/pubspec.yaml

```diff
@@ -27,3 +25,4 @@ dev_dependencies:
   file_testing: ^3.0.0
   lints: ^2.0.0
   test: ^1.16.0
+  test_api: ^0.4.13
```

Comment:
@GaryQian I made the minimal changes to pick up the version of `matcher` (a transitive dependency) that fixes the roll, but do so many of the dependencies in this package actually need to be pinned, or is that just a legacy of this code coming from `flutter/flutter`? If the latter, we should make these all `^` range dependencies to make this package less fragile, otherwise this kind of roll breakage is going to keep happening.